scene/resources/3d/height_map_shape_3d.h

```diff
@@ -55,6 +56,8 @@ class HeightMapShape3D : public Shape3D {
 	int get_map_depth() const;
 	void set_map_data(Vector<real_t> p_new);
 	Vector<real_t> get_map_data() const;
+	void set_tile_size(real_t p_new);
```

Comment:
```suggestion
	void set_tile_size(real_t p_tile_size);
```
Use descriptive argument names (matching bound argument)